### ISSUE TITLE
User-Action: Information E-Mail

### ISSUE THREAD

**Issue by matrix630307** [joomla/joomla-cms#39774]
### Steps to reproduce the issue
Update Joomla! 4.2.6 to 4.2.7
I have PHP Version 8.0.27

### Expected result
With 4.2.6 i have had (for example by login to a user) this Mail:

Letzte Benutzeraktivitäten
2023-01-25 15:52:45 UTC - Benutzer: Kampfrichter
Bereich der Aktivität: Benutzer
Meldung:
Benutzer <a href='https://www.pontonier.ch/cms/administrator/index.php?option=com_users&task=user.edit&id=155'>Kampfrichter</a> hat sich in der Website angemeldet

### Actual result
With 4.2.7 i become only this:

Letzte Benutzeraktivitäten

All Information after "Letzte Benutzeraktivitäten" will not delivered.
When i reset the Mail Template and insert only (for testing) the command: {DATE}
In the Mail i received only the text "{DATE}" - also no date - only this text.

### System information (as much as possible)
PHP 8.0.27
Joomla! (Updated from 4.2.6) 4.2.7

### Additional comments
In the Backend on Template i will see: com_actionlogs.notification

**Comment by matrix630307** [comment]
I have this Problem on many Sites (ca. 25 Websites). All the same Problem after Update.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39774">issues.joomla.org/tracker/joomla-cms/39774</a>.</sub>

**Comment by matrix630307** [comment]
I have activated this function on a new site (updated from 4.2.6 to 4.2.7) with the original Action-Template. I have activated the action "User". When i made some things in the backend, then i will become this message:

Letzte Benutzeraktivitäten
------
Dies ist die letzte Aktivität eines Benutzers auf der Website.

Aktivität | Datum | Erweiterung | Name

But under "Aktivität", "Datum" etc. i will no become data.
The Template is this:

Letzte Benutzeraktivitäten
------
Dies ist die letzte Aktivität eines Benutzers auf der Website.

Aktivität | Datum | Erweiterung | Name
{MESSAGES} {MESSAGE} | {DATE} | {EXTENSION} | {USERNAME} 
{/MESSAGES}<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39774">issues.joomla.org/tracker/joomla-cms/39774</a>.</sub>

**Comment by matrix630307** [comment]
@Philippe-FRICH Thank you for your confirmation. I hope, that the problem will solved in near time....

**Comment by jobrusche** [comment]
Still not solved.
Any hints what should be put in the mail template to make it work ?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39774">issues.joomla.org/tracker/joomla-cms/39774</a>.</sub>

**Comment by ReLater** [comment]
It's this change: https://github.com/joomla/joomla-cms/pull/38305/files

**Comment by Philippe-FRICH** [comment]
I submitted the "MailTemplate.php" file from joomla 4.2.6.
It does not change anything, there is always the same problem in the emails received.
{MESSAGE}{DATE}{EXTENSION}{USERNAME}

**Comment by ReLater** [comment]
> I submitted the "MailTemplate.php" file from joomla 4.2.6.

I have only exchanged the 3 or 4 lines of the pr, not the whole file.

I have tested with J!4.3-dev and the original mail template.

I received the email like 2 comments above.

Before: Like described in post https://github.com/joomla/joomla-cms/issues/39774#issuecomment-1413233068.

There the output after headlines is just empty (which I can confirm) 

and not

`{MESSAGE}{DATE}{EXTENSION}{USERNAME}`
